metricbeat/docs/modules/redis.asciidoc

```diff
@@ -63,9 +63,9 @@ metricbeat.modules:
   #maxconn: 10
 
   # Filters can be used to reduce the number of fields sent.
-  #filters:
+  #processors:
   #  - include_fields:
-  #      fields: ["stats"]
+  #      fields: ["beat", "metricset", "redis.info.stats"]
```

Comment:
The publisher pipeline right now applies processors after having constructed the complete event. The disadvantage is, when using `include_fields`, one must configure all event fields. Having to configure the full field name to keep is quite an advantage I think, as we also document the full field names in our reference.

Considering a change in the processors order, by applying the `processors` before adding beats fields like `beat` or `metricset`. For metricbeat this would require some refactoring such that each metricbeat has it's own beat.Client instance (right now all metricsets share one beat.Client by forwarding events to another go-routine). Then constant fields like `metricset` can be added by the publisher pipeline by configuring the `ClientConfig.Fields = common.MapStr { "metricset": ... }`.